orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -808,6 +808,11 @@ void addObjectMethodsForTraining(py::module& m, ExecutionProviderRegistrationFn
       })
       .def("save", [](PyGradientGraphBuilder* gradient_graph_builder, const std::string& path) {
         ORT_THROW_IF_ERROR(Model::Save(*(gradient_graph_builder->model), path));
+      })
+      .def("get_model", [](PyGradientGraphBuilder* gradient_graph_builder) {
+        std::string model_str;
+        gradient_graph_builder->model->ToProto().SerializeToString(&model_str);
```

Comment:
will this be fine to handle those large onnx model files?

Reply:
yes, this should be ok for large models. We do the same for `ORTModule` as well (which has worked fine for large models so far).